docs/linux/external_fuzzing_usbip.md

```diff
@@ -0,0 +1,34 @@
+# **USB/IP Fuzzing for Linux Kernel**
+
+Syzkaller supports fuzzing the Linux kernel USB/IP subsystem externally. We can set up a virtual network and send USB/IP packets to the client kernel as they are being received from an external server.  
+
+USB/IP fuzzing needs USB/IP configurations to be enabled. You can find the list in the configurations part. 
+
+USB/IP Fuzzing consists of 3 main parts:
+1. USB/IP pseudo-syscalls
+2. Syzkaller descriptions.
+3. KCOV changes that allow to collect coverage from background threads and interrupts
+
+### **Configurations:**
+
+Following configurations should be enabled and set for USB/IP. 
+`CONFIG_USBIP_VHCI_HC_PORTS` can be set from 1:15 with default value 8.
+`CONFIG_USBIP_VHCI_NR_HCS` can be set from 1:128 with default value 1.
```

Comment:
I don't think mentioning these ranges is needed. Just say:

`The following config options should be enabled or USB/IP.`